### ISSUE TITLE
Add `top` to Queryable

### ISSUE THREAD

**Issue by paulcsmith** [luckyframework/avram#340]
Similar to https://github.com/ankane/hightop

But with the Lucky spin on things:

```
VisitQuery.new.top(&.browser)
```

**Comment by wontruefree** [comment]
Is this `group_by` sorted by value?

**Comment by paulcsmith** [comment]
Yeah it groups items by the count of the given column, with the highest first. The examples in the linked Ruby gem show some helpful examples
